.github/workflows/linux_riscv.yml

```diff
@@ -0,0 +1,188 @@
+name: Linux RISC-V with Conan (Ubuntu 22.04, Python 3.10)
+on:
+  schedule:
+    # at 00:00 on Wednesday and Saturday
+    - cron: '0 0 * * 3,6'
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '**/docs/**'
+      - 'docs/**'
+      - '**/**.md'
+      - '**.md'
+      - '**/layer_tests_summary/**'
+      - '**/conformance/**'
+  push:
+    paths-ignore:
+      - '**/docs/**'
+      - 'docs/**'
+      - '**/**.md'
+      - '**.md'
+      - '**/layer_tests_summary/**'
+      - '**/conformance/**'
+    branches:
+      - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux-riscv
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores
+    container:
+      image: ubuntu:22.04
+      volumes:
+        - /mount/caches:/mount/caches
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      CMAKE_C_COMPILER_LAUNCHER: ccache
+      OPENVINO_REPO: /__w/openvino/openvino/openvino
+      OPENVINO_BUILD_DIR: /__w/openvino/openvino/openvino_build
+      INSTALL_DIR: /__w/openvino/openvino/openvino_install
+      CONAN_USER_HOME: /mount/caches/ccache/ubuntu22_riscv64_Release/.conan
+      CCACHE_DIR: /mount/caches/ccache/ubuntu22_riscv64_Release
+      CCACHE_TEMPDIR: /__w/openvino/openvino/ccache_temp
+      CCACHE_MAXSIZE: 50G
+    steps:
+      - name: Install git
+        run: apt-get update && apt-get install --assume-yes --no-install-recommends git ca-certificates
+
+      - name: Clone OpenVINO
+        uses: actions/checkout@v4
+        with:
+          path: 'openvino'
+
+      - name: Init submodules for non-Conan dependencies
+        run: |
+          pushd ${OPENVINO_REPO}
+            git submodule update --init -- ${OPENVINO_REPO}/src/plugins/intel_cpu
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/gtest
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/open_model_zoo
+          popd
+
+      #
+      # Dependencies
+      #
+
+      - name: Install build dependencies
+        run: |
+          # create build directory
+          mkdir -p ${OPENVINO_BUILD_DIR}
+
+          # install compilers to build OpenVINO for RISC-V 64
+          apt --assume-yes install gcc-riscv64-linux-gnu g++-riscv64-linux-gnu
+          apt --assume-yes install gcc g++ python3-pip python3-venv python3-dev
+          # generic dependencies
+          apt --assume-yes install cmake ccache ninja-build fdupes patchelf
+
+          python3 -m venv ${OPENVINO_BUILD_DIR}/env
```

Comment:
Why not `setup-python`? System py3.10 is enough? 

The output (`python-path`) from the `setup-python` action could be used, it points to the Python executable: https://github.com/actions/setup-python/blob/main/action.yml#L34, there would be no need to create venv.

Reply:
I use both system python for x86_64 and riscv64 for consistency